### ISSUE TITLE
Bitwarden "drag tool here"

### ISSUE THREAD

**Issue by notaduck** [minbrowser/min#965]
I recently installed the `min` browser in the hope of replacing firefox. However, I'm facing some issues with bitwarden,It says that I've to drag the cli tool. But I don't have any file managers to do so?! Is there another way?

**Comment by PalmerAL** [comment]
Hmm, I guess we should add an option to click on the input and show a file dialog as well. In the meantime, I think you should be able to click the link to download the CLI tool in min, and then drag the download from the bar at the bottom into the box.

**Comment by notaduck** [comment]
Thanks for the quick reply, I thing the option to open a file dialog to choose the file would be a good opthen. However, the suggested way of doing it worked! Thanks.

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by q-codes** [comment]
Where do I find the cli tool? I think, there are only .AppImage and .deb

**Comment by notaduck** [comment]
Have you tried to click on the link?

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by q-codes** [comment]
Yes, from Debian 10, the link downloads bitwarden for windows. I tried cli from github. But, it is taking forever to install. Maybe, my root FS is broken. But, cli from GitHub shall and should work.

**Comment by PalmerAL** [comment]
Oops, it looks like the link is broken on Linux (although it sounds like it worked for you @notaduck ?).

Anyway, I think you can use this link: https://vault.bitwarden.com/download/?app=cli&platform=linux and install the CLI globally, then after you restart Min it should be detected automatically.

**Comment by q-codes** [comment]
`Failed to install...username or password is incorrect....Make sure you use the right file`

But, I am using [https://vault.bitwarden.com/download/?app=cli&platform=linux](url)

**Comment by notaduck** [comment]
> Oops, it looks like the link is broken on Linux (although it sounds like it worked for you @notaduck ?).
> 
> Anyway, I think you can use this link: https://vault.bitwarden.com/download/?app=cli&platform=linux and install the CLI globally, then after you restart Min it should be detected automatically.

It worked fine for me :) I've used bitwarden the last couple of days, and it works really well. However it would be nice to be able to generate and add passwords to my acc.
